app/models/cache/IleQuery.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.cache
+
+import java.time.{Instant, ZoneOffset}
+
+import play.api.libs.json.{JsError, JsNumber, JsObject, JsResult, JsSuccess, JsValue, Json, OFormat}
+
+case class IleQuery(sessionId: String, ucr: String, conversationId: String, createdAt: Instant = Instant.now())
+
+object IleQuery {
```

Comment:
This has the same name as `forms.IleQuery`. I'd change one of them.

Reply:
yes, agree. What propose? IleQueryForm?

Reply:
Changed Form to have different name